### ISSUE TITLE
Expose `cc.xml` without authentication

### ISSUE THREAD

**Issue by jzimermann** [concourse/concourse#3979]
## What challenge are you facing?
Recently, version 5.x addressed the issue #438, exposing the `cc.xml` file. Currently checking the feature, it seems this endpoint is under authentication, depending upon a cookie called `skymarshal_auth`.

## What would make this better?
Would it be possible to expose this endpoint without this needed authentication?

**Comment by vito** [comment]
Yeah, looks like it requires auth for the team. What would you want it to show when not logged in? Exposed pipelines?

**Comment by jzimermann** [comment]
@vito Would it possible to display at the least the pipelines that were exposed with the command `fly expose-pipeline`?

**Comment by vito** [comment]
Yep that makes sense to me.

I don't think we'll get around to this in the near-term but I would welcome a PR!

**Comment by jzimermann** [comment]
Would be possible to include this as part of the next releases?

By delivering this feature, the teams that I work with will be able to have a better monitoring of our builds, using information radiators, like [NeverGreen](https://nevergreen.io)

**Comment by vito** [comment]
We rely on community contributions for odds and ends like this as we really don't have the bandwidth to chase down every feature ourselves. Sorry. :slightly_smiling_face: The fastest way to get this done would be via a PR - check out [`CONTRIBUTING.md`](https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md) if you're interested!
